scripts/install-jdk.sh

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env bash
+
+#
+# Install JDK for Linux
+#
+# This script determines the most recent early-access build number,
+# downloads the JDK archive to the user home directory and extracts
+# it there.
+#
+# Example usage
+#
+#   install-jdk.sh                 | install most recent (early-access) JDK
+#   install-jdk.sh -W /usr/opt     | install most recent (early-access) JDK to /usr/opt
+#   install-jdk.sh -F 9            | install most recent OpenJDK 9
+#   install-jdk.sh -F 10           | install most recent OpenJDK 10
+#   install-jdk.sh -F 10 -L BCL    | install most recent OracleJDK 10
+#   install-jdk.sh -F 11           | install most recent OpenJDK 10
```

Comment:
comment should be updated to OpenJDK 11 as well.

Reply:
Dooh. Done. Force-pushed changes.